runtime/src/lib.rs

```diff
@@ -265,10 +265,13 @@ impl Contains<Call> for BaseFilter {
 				Call::Vesting(pallet_vesting::Call::vest { .. }) |
 				Call::Vesting(pallet_vesting::Call::vest_other { .. })
 			);
+		let is_domains_registration =
+			matches!(c, Call::Domains(pallet_domains::Call::register_domain { .. }));
```

Comment:
Why?

Reply:
We need to have registration disabled until all current handles from solochain and reserved words imported